### PR TITLE
feat: Add prefix config to directory module

### PR DESCRIPTION
**Pull request by wendorf**
#### Description
Prefix is customizable for the directory module

I understand that #624 gets rid of the need for this configuration, but I only found it after already making the change. If #624 will take a while to complete, it may still be useful to merge this PR.

#### Motivation and Context
For my prompt config, the directory is obvious and a prefix is not necessary. This helps clean that up.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?
- [x] I have tested using **MacOS**
- [x] I have tested using **Linux**
- [ ] I have tested using **Windows**

* Unit test that passes in the new configuration and shows that it affects rendering
* Manually tested by setting the configuration and running `eval "$(./target/debug/starship init bash)"` with a binary that includes this PR
* `./acceptance_test`.

#### Checklist:
- [x] I have updated the documentation accordingly.
- [x] I have updated the tests accordingly.

### CODE REVIEW

File: docs/config/README.md [hunk 1]
```diff
@@ -327,6 +327,7 @@ it would have been `nixpkgs/pkgs`.
 | `truncate_to_repo`  | `true`        | Whether or not to truncate to the root of the git repo that you're currently in. |
 | `style`             | `"bold cyan"` | The style for the module.                                                        |
 | `disabled`          | `false`       | Disables the `directory` module.                                                 |
+| `prefix`            | `"in "`       | Prefix to display immediately before the directory.                              |
```
Comment (andytom) [inline docs/config/README.md]: I think the convention is to put the `prefix` above the `style` and `disabled` variables.

Comment (wendorf) [reply docs/config/README.md]: Thanks for the note. Just pushed an update to move it up.

**Comment by alexjurkiewicz** [comment]
Any chance this can be merged? It's a great feature

**Comment by matchai** [comment]
Thank you for your contribution @wendorf! 🎉 
@all-contributors Please add @wendorf for code and test.

**Pull request merged**